moveit_plugins/moveit_ros_control_interface/src/controller_manager_plugin.cpp

```diff
@@ -72,6 +72,19 @@ bool checkTimeout(ros::Time &t, double timeout, bool force = false)
   return false;
 }
 
+namespace
+{
+  /**
+   * @brief Convert a boost::shared_ptr to an std::shared_ptr as good as possible.
+   * This is a dangerous conversion that only works in specific cases. Do not use blindly!
+   */
+  template<typename T>
+  std::shared_ptr<T> convert_shared_ptr(boost::shared_ptr<T> && ptr)
+  {
+    return std::shared_ptr<T>(ptr.get(), [ptr](T*) mutable {ptr.reset();});
+  }
+}
```

Comment:
ros/class_loader#38 is merged, ros/pluginlib#42 is the last missing piece. Then the only thing remaining for deleting these functions is waiting for their releases.